spec/lib/tasks/user_spec.rb

```diff
@@ -44,6 +54,35 @@
         so their approval status has been set to "not_approved".
       MESSAGE
 
+      allow(Rails.logger).to receive(:info)
+      expect(Rails.logger).to receive(:info).with(expected_message)
+      @rake[task_name].invoke
+    end
+  end
+
+  describe 'usasearch:user:update_not_active_approval_status' do
```

Comment:
Looks like you've accidentally duplicated the entire `describe 'usasearch:user:update_not_active_approval_status'` block.

Reply:
GAHHH!!